packages/react-storage/src/components/StorageBrowser/tasks/types.ts

```diff
@@ -0,0 +1,31 @@
+import { TaskHandlerInput } from '../actions';
+
+export type TaskStatus =
+  | 'CANCELED'
+  | 'FAILED'
+  | 'COMPLETE'
+  | 'QUEUED'
+  | 'PENDING';
+
+export interface ProcessTasksOptions {
+  concurrency?: number;
+  isCancelable?: boolean;
+}
+
+export interface Task<T = any> {
+  cancel: undefined | (() => void);
+  item: T;
+  key: string;
+  message: string | undefined;
```

Comment:
can this be `message?: string` ?

Reply:
Output values should not be optional unless defined by the consumer